src/core/components/hv-root/index.tsx

```diff
@@ -503,6 +521,11 @@ export default class Hyperview extends PureComponent<Types.Props> {
     const behavior = this.behaviorRegistry[action];
 
     if (Behaviors.isOncePreviouslyApplied(behaviorElement)) {
+      Logging.info(
+        `[behavior] ${action} on ${behaviorElement?.getAttribute(
```

Comment:
this is going to do an extra DOM look up, you probably should move the call to getAttribute to your new deferred to string function

Reply:
Thanks! I was on the fence and didn't want to go overboard. I'll do another pass to look for other areas like this.